pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -607,7 +607,11 @@ ASTFormalParameter FormalParameter()  :
    (
 	simpleNode = ID()
 	// the order of outer "|" is important !
-	( LOOKAHEAD(2) ( LOOKAHEAD(2) ( (<OUT>|(<IN> <OUT>)) (LOOKAHEAD(2) <NOCOPY>  )? ) | <IN> ) )?
+
+    (<IN> {jjtThis.setIn(true); } )?
+    (<OUT> {jjtThis.setOut(true); } )?
+    (<NOCOPY> {jjtThis.setNoCopy(true); } )?
```

Comment:
This looks correct (see https://docs.oracle.com/en/database/oracle/oracle-database/18/lnpls/formal-parameter-declaration.html#GUID-5BA8E033-96B9-439A-A4FC-4844FEC14AD8). The previous version was much more confusing. We can remove now the comment `//the order of outer "|" is important !`